examples/apps/mocks/commerce-mock.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: v1
```

Comment:
This is also a very SAP focused example. Most probably should not stay in the examples